internal/auth/jwt.go

```diff
@@ -16,10 +29,42 @@ func NewTokenManager(publicKey, privateKey string) *TokenManager {
 
 // GenerateAccessToken generates new access token.
 func (tm *TokenManager) GenerateAccessToken(userID string) (string, error) {
-	return "", nil
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.StandardClaims{
+		Subject:   userID,
+		ExpiresAt: time.Now().Add(ACTimeout).Unix(),
+		IssuedAt:  time.Now().Unix(),
+	})
+
+	return token.SignedString(tm.privateKey)
 }
 
 // GenerateRefreshToken generates new refresh token.
 func (tm *TokenManager) GenerateRefreshToken() (string, error) {
-	return "", nil
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.StandardClaims{
+		ExpiresAt: time.Now().Add(RTTimeout).Unix(),
+		IssuedAt:  time.Now().Unix(),
+	})
+
+	return token.SignedString(tm.privateKey)
+}
+
+//ParseToken parses the given token and returns the user ID.
```

Comment:
I guess this method is not only parsing the token, but also validating it. Let's add a check for token expiration